launch_ros_sandbox/actions/load_docker_nodes.py

```diff
@@ -194,9 +195,25 @@ def _handle_logs(
         all of the nodes.
 
         """
-        self._pull_docker_image()
+        # Try to pull the image and warn if it cannot be found.
+        try:
+            self._pull_docker_image()
+        except ImageNotFound as ex:
+            self.__logger.warn(ex)
```

Comment:
Can you log here something like "image could not be pulled, but it may be present locally, trying to run", so that the warning is very clear to the user?

Reply:
Sure.